docker-compose.yml

```diff
@@ -19,6 +21,8 @@ services:
     environment:
       - MONGO_URI=mongodb://mongo:27017/cheburashka
       - SECRET_OR_KEY=cheburashka
+    links:
```

Comment:
I think it's not necessary

Reply:
Same as previous comments - this change is 2 lines not 1, and it's needed so "api" service will be able to communicate with "mongo" service.